arcgis-ios-sdk-samples/Layers/Query with CQL filters/QueryWithCQLFiltersViewController.swift

```diff
@@ -0,0 +1,115 @@
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            let map = AGSMap(basemapStyle: .arcGISTopographic)
+            map.operationalLayers.add(ogcFeatureLayer)
+            mapView.map = map
+            // Set the viewpoint to Daraa, Syria.
+            mapView.setViewpoint(AGSViewpoint(latitude: 32.62, longitude: 36.10, scale: 20_000))
+        }
+    }
+    /// The button to present the CQL filters settings.
+    @IBOutlet var cqlFiltersBarButtonItem: UIBarButtonItem!
+    
+    // MARK: Properties
+    
+    /// The most recent query job.
+    var lastQuery: AGSCancelable?
+    
+    /// A feature layer to visualize the OGC API features.
+    let ogcFeatureLayer: AGSFeatureLayer = {
+        // Note: the collection ID can be accessed later via
+        // `featureCollectionInfo.collectionID` property of the feature table.
+        let table = AGSOGCFeatureCollectionTable(
+            url: URL(string: "https://demo.ldproxy.net/daraa")!,
+            collectionID: "TransportationGroundCrv"
+        )
+        // Set the feature request mode to manual (only manual is currently
+        // supported). In this mode, you must manually populate the table -
+        // panning and zooming won't request features automatically.
+        table.featureRequestMode = .manualCache
+        
+        let featureLayer = AGSFeatureLayer(featureTable: table)
+        let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .red, width: 3)
+        featureLayer.renderer = AGSSimpleRenderer(symbol: lineSymbol)
+        return featureLayer
+    }()
+    
+    // MARK: Method
+    
+    func populateFeaturesFromQuery(queryParameters: AGSQueryParameters) {
+        // Cancel if there is an existing query request.
+        lastQuery?.cancel()
+        
+        // Populate the table with the query. Setting `outFields` to `nil`
+        // requests all fields.
+        let table = ogcFeatureLayer.featureTable as! AGSOGCFeatureCollectionTable
+        lastQuery = table.populateFromService(
+            with: queryParameters,
+            clearCache: true,
+            outfields: nil
+        ) { [weak self] result, error in
+            guard let self = self else { return }
+            self.lastQuery = nil
+            if let error = error,
+               // Do not display error if user cancelled the request.
+               (error as NSError).code != NSUserCancelledError {
+                self.presentAlert(error: error)
+            } else if let result = result, let extent = self.ogcFeatureLayer.featureTable?.extent {
+                // Zoom to the extent of the selected collection.
+                self.mapView.setViewpointGeometry(extent, padding: 50)
+                self.presentAlert(title: "Query Result", message: "Query returned \(result.featureEnumerator().allObjects.count) features.")
```

Comment:
Consider using number formatter so that count is formatted properly, such as using a group separator when greater than `999`.

```suggestion
                self.presentAlert(title: "Query Result", message: "Query returned \(NumberFormatter.localizedString(from: result.featureEnumerator().allObjects.count as NSNumber, number: .decimal)) features.")
```